app/styles/ui/toolbar/_toolbar.scss

```diff
@@ -33,19 +33,20 @@
 
   .toolbar-button {
     &.push-pull-button {
-      width: 230px;
+      width: 100%;
     }
   }
 
   .toolbar-button {
     &.branch-toolbar-button {
-      width: 230px;
+      width: 100%;
```

Comment:
Apply flexible width on the Branch and Push/Pull toolbar buttons.

Reply:
There’s another button just below named `.revert-progress` also with fixed width.

```
&.revert-progress {
    width: 230px;
}
```

I believe this button should also change to `width: 100%;` but haven’t yet found a way to get to a state where the button shows.

Reply:
This is seen when reverting a commit in a git LFS repo in which reverting a commit is very slow. Thus, this does need addressed. We can follow up with another PR since Git LFS is not a common usage and the likely hood that this would result in a negative experience is slim (worse case during loading the last toolbar button switches to 230px which could only be a clipping problem on a very small zoomed in screen). The `RevertProgress` toolbar button is rendered instead of the `PullPushFetch` so we can send the dimensions for the `PullPushFetch`  button down to it.